[34m# test 1[39m
[32mok 1 should be equal[39m
[32mok 2 should be equal[39m
[34m# test 2[39m
[31mnot ok 3 Error: This is an error[39m
    type:    Error
    message: This is an error
    code:    EINVAL
    errno:   ~
    file:    /node/taper/test/throws.js
    line:    10
    column:  9
    stack:   
      - Test.<anonymous> (/node/taper/test/throws.js:10:9)
      - Test.emit (events.js:117:20)
      - Test.emit (/node/taper/node_modules/tap/lib/tap-test.js:103:8)
      - GlobalHarness.Harness.process (/node/taper/node_modules/tap/lib/tap-harness.js:86:13)
      - process._tickCallback (node.js:419:13)
      - Function.Module.runMain (module.js:499:11)
      - startup (node.js:119:16)
      - node.js:906:3
    thrown:  true
  ...

[31mError thrown: Error "This is an error" 
	code: EINVAL, errno: null
[39m  Test.<anonymous> (/node/taper/test/throws.js:10:9)
  Test.emit (events.js:117:20)
  Test.emit (/node/taper/node_modules/tap/lib/tap-test.js:103:8)
  GlobalHarness.Harness.process (/node/taper/node_modules/tap/lib/tap-harness.js:86:13)
  process._tickCallback (node.js:419:13)
  Function.Module.runMain (module.js:499:11)
  startup (node.js:119:16)
  node.js:906:3

